controllers/datamovement_controller_test.go

```diff
@@ -91,7 +197,288 @@ var _ = Describe("Data Movement Test" /*Ordered, (Ginkgo v2)*/, func() {
 		})
 	})
 
+	Context("when the dm configmap has specified a dmCommand", func() {
+		BeforeEach(func() {
+			setDefaultConfigValues()
+			cmData[DMConfigKeyCmd] = "ls -l"
+		})
+		It("should use that command instead of the default mpirun", func() {
+			Eventually(func(g Gomega) string {
+				cmd := ""
+				g.Expect(k8sClient.Get(context.TODO(), client.ObjectKeyFromObject(dm), dm)).To(Succeed())
+				if dm.Status.CommandStatus != nil {
+					cmd = dm.Status.CommandStatus.Command
+				}
+				return cmd
+			}).Should(Equal("/bin/" + cmData[DMConfigKeyCmd]))
```

Comment:
Where does the`/bin` come from? I thought this was passed and run as is.

Reply:
Normally it is, but when you use `sleep`, the `/bin/` gets prepended to it.

Reply:
That's really strange, I guess I wouldn't expect that. Are we explicit about the command then (`/bin/ls -l`)? Or just leave it as is?

Reply:
I'll be explicit.